src/es2015/text.js

```diff
@@ -0,0 +1,4 @@
+module.exports = () => {
+	// https://mathiasbynens.be/notes/es-unicode-property-escapes#emoji
+	return /<% emojiSequenceUnicode %>|\p{Emoji_Modifier_Base}\p{Emoji_Modifier}?|\p{Emoji_Presentation}|\p{Emoji}\uFE0F?/gu;
+};
```

Comment:
Looks like this can be done, but it requires two Babel passes, and separating configs by “env”!